### DISCUSSION TITLE
Cannot stringify a function Issue

### DISCUSSION THREAD

**Discussion by islamzaoui** [sveltejs/kit#10480] [Q&A]
Hello guys
I want to load the markdown file content to the page but I get this error:
`Error: Data returned from 'load' while rendering /courses/[module]/[lesson] is not serializable: Cannot stringify a function (data.lesson.content.render)`
this is my code:
```
export const load = (async (event) => {
	const Lesson = await import(`../../../../../Lessons/${event.params.module}/${event.params.lesson}.md`)
	return { 
		metadata: Lesson.metadata, 
		content: Lesson.default 
	}
});
```
the problem with the `Lesson.default ` have this  ` { render: [Function: render], '$$render': [Function: $$render] } `
is there any fix to this problem?

**Comment by david-plugge** [comment]
If you want the actual content you can add `?raw` to the import path
